### PR TITLE
HDDS-10248. Remove unused, dead code in hdds-client module

### PR DESCRIPTION
**Pull request by fapifta**
## What changes were proposed in this pull request?
Removing dead code from hdds-client module. Dead code besides that it is not used can be harmful, if someone starts to use it, as it was not used it might not have been updated either along with similar functionality, so it is better to get rid of it.

Please describe your PR in detail:
The PR is, removing unused code from hdds-client module.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-10248

## How was this patch tested?
Functionality should not be changing, if the code compiles, and existing tests are running fine, that pretty much proves that we did not needed these methods.
On the other hand, due to potential reflective access it is hard to prove that complete methods are not called, therefore I ran a search for the method name in the codebase and checked for that manually.

### CODE REVIEW

**Comment by kerneltime** [comment]
Minor cleanup needed 
```
hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java
 21: Unused import - java.time.ZoneId.
 22: Unused import - java.time.format.DateTimeFormatter.
```

**Comment by adoroszlai** [comment]
@fapifta please wait for clean CI run in your fork before opening PR

File: hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java [hunk 1]
```diff
@@ -18,10 +18,7 @@
 
 package org.apache.hadoop.hdds.scm.client;
 
-import java.text.ParseException;
-import java.time.Instant;
 import java.time.ZoneId;
```
Comment (adoroszlai) [inline hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java:21]:
```suggestion
```

File: hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java [hunk 2]
```diff
@@ -18,10 +18,7 @@
...
-import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
```
Comment (adoroszlai) [inline hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java:22]:
```suggestion
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @fapifta for working on this.  (Suggesting trivial code changes pointed out by checkstyle and @kerneltime, only to be able to update the PR and get CI running.)

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @fapifta for the cleanup, @kerneltime for the review.

**Pull request merged**